.mailmap

```diff
@@ -382,6 +382,10 @@ Oluwaseun Omoyajowo <omoyajowo2015@gmail.com>
 OneNail <OneNail@yeah.net> <onenail@yeah.net>
 Onne Gorter <onne@onnlucky.com>
 Oscar Martinez <oscar@mtnz-web.com> <oscar.martinez@hautelook.com>
+Paolo Insogna <paolo@cowtech.it>
+Paolo Insogna <paolo@cowtech.it> <paolo.insogna@nearform.com>
+Paolo Insogna <paolo@cowtech.it> <shogun@cowtech.it>
+Paolo Insogna <paolo@cowtech.it> <ShogunPanda@users.noreply.github.com>
```

Comment:
These lines aren't needed unless you plan to commit from those email addresses in the future. 

```suggestion
```